README.md

```diff
@@ -94,6 +97,13 @@ if __name__ == '__main__':
     print("Contradiction:", predicted_probability[2])
 ```
 
+If you are using our pre-trained model checkpoints with the above code snippet, you would expect to got the following numbers.
+Huggingface Model Hub Checkpoint | A1 (dev) | A2 (dev) | A3 (dev)
+---|---|---|---
+ynie/roberta-large-snli_mnli_fever_anli_R1_R2_R3-nli | 73.8 | 50.8 | 46.1
+ynie/xlnet-large-cased-snli_mnli_fever_anli_R1_R2_R3-nli | 73.4 | 52.3 | 50.8
+ynie/albert-xxlarge-v2-snli_mnli_fever_anli_R1_R2_R3-nli | 76.0 | 57.0 | 57.0
+
```

Comment:
What about test numbers?

Reply:
I was thinking we can reserve the test numbers here. We can show them if you prefer.